.github/workflows/macos-release.yml

```
@@ -39,12 +39,13 @@ jobs:
       - name: Build Backend
         run: |
           go mod download
-          go build -ldflags "-X 'one-api/common.Version=$(git describe --tags)'" -o new-api-macos
+          VERSION=$(git describe --tags)
+          go build -ldflags "-X 'one-api/common.Version=$VERSION'" -o new-api-macos-$VERSION
       - name: Release
         uses: softprops/action-gh-release@v1
```

actionlint (1.7.7)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)